web/docs/guides/with-nuxt-3.mdx

```diff
@@ -0,0 +1,33 @@
+---
+id: with-nuxt-3
+title: "Quickstart: Nuxt 3"
+description: Learn how to use Supabase in your Nuxt 3 App.
+---
+
+## Intro 
+
+This example provides the steps to build a simple todo app (from scratch!) using Supabase and Nuxt 3. It includes:
+
+- Supabase [Database](/docs/guides/database): a Postgres database for storing your user data.
+- Supabase [Auth](/docs/guides/auth): users can sign in with github provider.
+- Instant [APIs](/docs/guides/api): APIs will be automatically generated when you create your database tables.
+
+By the end of this guide you'll have an app which allows users to login and use our supabase hosted todo list.
+
+[![n3-supabase-demo](https://user-images.githubusercontent.com/904724/160422461-8f87500a-8dec-4413-86b2-ba04e1b2d17b.png)](https://n3-supabase.netlify.app)
+
+## Live demo
+
+You can check a [live demo](https://n3-supabase.netlify.app) of this project.
+
+## Do it yourself
+
+
+Just follow the [readme on Github](https://github.com/nuxt-community/supabase-module/tree/main/demo#todo-list-example-using-supabase-and-nuxt-3) to see the app in action.
```

Comment:
although admittedly this repo is very well documented, the intent of these guides is to have a "build it yourself" so that users can run through step-by-step and build from scratch to learn _how_. If it's just a "deploy", then they don't learn the ins-and-outs of your framework & Supabase

Perhaps this one should be linked as a resource?

Reply:
What if we make it a step by step guide on https://supabase.nuxtjs.org/tutorial teaching how to make our demo? We could align ours to your (magic link + file upload)

Reply:
I removed the guide for the moment. I'll create the exact same stem by step tutorial later. Is it ok for you @kiwicopple ?